### PR TITLE
add configurable environment value

### PR DESCRIPTION
**Pull request by real-danm**
This sets a config.Environment value. It will also label prometheus metrics with this value. The default is empty.

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ type Config struct {
 	Port           uint32                   `yaml:"port"`
 	BindAddresses  []string                 `yaml:"bind_addresses"`
 	PrometheusPort uint32                   `yaml:"prometheus_port,omitempty"`
+	PrometheusEnv  string                   `yaml:"prometheus_env,omitempty"`
```
Comment (matkam) [inline pkg/config/config.go:49]:
```suggestion
	Environment  string                   `yaml:"environment,omitempty"`
```

File: cmd/server/main.go [hunk 2]
```diff
@@ -250,7 +250,15 @@ func startServer(c *cli.Context) error {
...
-	prometheus.Init(currentNode.Id, currentNode.Type)
+	var promEnv string
+	if conf.PrometheusEnv != "" {
+		promEnv = conf.PrometheusEnv
+	} else if conf.Development {
+		promEnv = "dev"
+	} else {
+		promEnv = "prod"
+	}
+	prometheus.Init(currentNode.Id, currentNode.Type, promEnv)
```
Comment (matkam) [inline cmd/server/main.go:261]:
```suggestion
	prometheus.Init(currentNode.Id, currentNode.Type, conf.Environment)
```

Comment (matkam) [reply cmd/server/main.go:261]: to keep things in one place, put this in `config.go` `NewConfig(...)`:
```
if conf.Development {
  conf.Environment = "dev"
}
```

And I think its fine to leave the default as "" and not assume the environment name.

File: config-sample.yaml [hunk 3]
```diff
@@ -110,6 +110,8 @@ rtc:
 
 # when enabled, LiveKit will expose prometheus metrics on :6789/metrics
 # prometheus_port: 6789
+# tag prometheus metrics with custom env label; defaults to dev|prod
+# prometheus_env: custom-label
```
Comment (matkam) [inline config-sample.yaml:114]:
```suggestion
# tag prometheus metrics with custom env label; defaults to empty string
# environment: custom-label
```

**Review by matkam — CHANGES_REQUESTED** [review]

**Comment by boks1971** [comment]
> This tags prometheus metrics with a configurable 'env' label. The default is either dev or prod.

Maybe edit this to mention default is empty.

**Review by boks1971 — APPROVED** [review]

**Review by matkam — APPROVED** [review]

**Pull request merged**